package.json

```diff
@@ -84,12 +81,14 @@
     "@storybook/addons": "5.0.5",
     "@storybook/react": "5.0.5",
     "@storybook/storybook-deployer": "2.8.1",
+    "@types/chartist": "^0.9.46",
     "@types/enzyme": "3.1.14",
     "@types/jest": "23.3.5",
     "@types/memoize-one": "3.1.1",
     "@types/node": "^10.12.26",
-    "@types/react": "16.4.16",
+    "@types/react": "16.9.23",
```

Comment:
tradeoff: we're using types for newer versions of react than our peerDeps state...

Reply:
That is a problem, we discussed increasing the react peer dependency but I can't recall what we decided on @mperrotti might have a better memory than I do.

Reply:
We talked about it in the last monthly frontend sync. We're ok with increasing the React peer dependency 👍

Reply:
Can we then add a newer peer dependency here as well.

Reply:
done

Reply:
Thanks, I was about to do it ;-) Thanks for  being faster then me.